buildSrc/src/main/java/Dependencies.kt

```diff
@@ -25,7 +25,7 @@ object Dependencies {
     const val coreKtx = "androidx.core:core-ktx:$coreKtxVersion"
 
     // Firebase
-    private const val firebaseConfigVersion = "19.1.4"
```

Comment:
how about using firebase bom?

Reply:
I think using BOM for one dependency is kinda overkill and not why it was made initially. It was made so devs can maintain their 100 firebase libs changing only one dependency version. There we will change 1 lib version one way or another